assets/src/styles/partials/_parameter-list.scss

```diff
@@ -0,0 +1,92 @@
+/*
+ * Styles related to the Parameter Selection List.
+ */
+@import '../components/hydrograph/variables';
+
+$row-border-color: 'black';
+
+.select-time-series-container {
+  #parameter-selection-title {
+    @include u-font-weight('bold');
+    @include u-font-size('body', 'lg');
+    @include u-margin-bottom(2px);
+  }
+
+  #select-time-series {
+    @include grid-container;
+    @include u-margin(0);
+    @include u-padding(0);
+  }
+
+  .main-parameter-selection-container {
+    .selected {
+      background-color: $selected;
+    }
+
+    .grid-row-period-of-record {
+      @include grid-row;
+      .grid-row-period-of-record-text {
+        @include grid-col(10);
+        @include grid-offset(1);
+      }
+      .toggle-for-top-period-of-record {
+        @include grid-col(1);
+      }
+      @include at-media('tablet') {
+        display: none;
+      }
+    }
+
+    .grid-row-container-row {
+      @include grid-container;
+      @include u-border-bottom(1px, $row-border-color);
+      @include u-padding(1);
+    }
+
+    .grid-row-container-row:first-child {
+      @include u-border-top(1px, $row-border-color);
+    }
+
+    .grid-row-inner {
+      .usa-radio__label {
+        @include u-margin-top(0);
+        @include u-margin-bottom(0);
+      }
+      .description__param-select {
+        @include grid-col(7);
+        @include u-font-weight('bold');
+      }
+      @include at-media('tablet') {
+        .description__param-select {
+          @include u-font-weight('normal');
+        }
+      }
+    }
+
+    .grid-row-inner {
+      @include u-padding-right(0);
+      @include u-padding-left(0);
+
+      .wateralert-row {
+        @include grid-col;
+        @include grid-offset(1);
+      }
+    }
+
+    .period-of-record__param-select {
+      @include grid-col;
+      display: none;
+      @include at-media('tablet') {
+        display: inline;
+      }
+    }
+
+    #period-of-record-and-toggle-container {
+      display: none;
+      @include at-media('tablet') {
+        display: flex;
```

Comment:
FYI: There are mixins for flex too, you might be able to use here: https://designsystem.digital.gov/utilities/flex/

As a philosophy thing, I like using the mixins because they just work, they are documented in the USWDS and combine one or more style rules. I think for developers who are less familiar with the details of CSS may find these easier to understand.

Reply:
This was another thing I was messing with and then forgot about. I have changed the 'flex' one, but I still can't figure out the mixin syntax for 'justify-content: space-between;' or 'display: none;'

Reply:
Sometimes going spelunking in the USWDS for the mixin reveals these kinds of secrets. But I am also fine with leaving as is.